ppr-ui/src/components/parties/party/PartyAutocomplete.vue

```diff
@@ -161,44 +144,18 @@ export default defineComponent({
         setRegisteringParty(newParty)
       } else if (props.isMhrPartySearch) {
         localState.selectedCode = newParty.code
-        if (props.isMhrTransfer) {
-          // pre-set country code to prevent clearing of base-address component fields
-          setMhrTransferSubmittingPartyKey({ key: 'address.country', value: newParty.address.country })
-          setMhrTransferSubmittingParty(newParty)
-        } else {
-          // pre-set country code to prevent clearing of base-address component fields (bug 13637)
-          setMhrSubmittingParty({ key: 'address.country', value: newParty.address.country })
-          setMhrRegistrationSubmittingParty(newParty)
-        }
```

Comment:
Handled in parent component -> see `handlePartySelect` in `ContactInformation.vue`